packages/renderer/src/lib/preferences/PreferencesAuthenticationProvidersRendering.spec.ts

```diff
@@ -232,12 +232,11 @@ test('Expects images.icon option to be used when no themes are present', async (
   authenticationProviders.set(providerWithImageIcon);
   render(PreferencesAuthenticationProvidersRendering, {});
 
-  // wait for image to be loaded
-  await new Promise(resolve => setTimeout(resolve, 200));
-
-  const icon = screen.getByRole('img', { name: `${testProvidersInfoWithSessionRequests[0].displayName}` });
-  expect(icon).toBeInTheDocument();
-  expect(icon).toHaveAttribute('src', './icon.png');
+  await waitFor(() => {
+    const icon = screen.getByRole('img', { name: `${testProvidersInfoWithSessionRequests[0].displayName}` });
+    expect(icon).toBeInTheDocument();
+    expect(icon).toHaveAttribute('src', './icon.png');
+  });
```

Comment:
suggestion: a nice synthax I like to use, is only keeping inside the wait synthax inside the waitFor


```suggestion
  const icon = await waitFor(() => {
    return screen.getByRole('img', { name: `${testProvidersInfoWithSessionRequests[0].displayName}` });
  });
    
  expect(icon).toBeInTheDocument();
  expect(icon).toHaveAttribute('src', './icon.png');
```

while reading, we clearly understand what we are waiting for, here (I guess,) that the img is available